docs/DeveloperGuide.md

```diff
@@ -4,10 +4,7 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
-
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
+## Class Structure
 
 ## Product scope
```

Comment:
Clearly specifying the product scope could help developers better understand the product. A more comprehensive outline of the user stories can provide context of the product, which allows better understanding of the key features and functionalities of your code